home/templatetags/format_metadata_field_type.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def format_metadata_field_type(value: dict) -> str:
+    if value.get("title"):
+        title = value["title"]
+        if value.get("type"):
+            type_string = value["type"]
+        if value.get("anyOf"):
+            type_string = "|".join(item["type"] for item in value["anyOf"])
```

Comment:
I noticed this displays `(string|null)` for optional strings. Could we make this a bit more user friendly, e.g. `(optional string)` or `(string or null)`